docsrc/Connecting_and_queries.rst

```diff
@@ -246,32 +246,110 @@ Multiple-statement queries allow you to run a series of SQL statements sequentia
 
 	cursor.close()
 
-**Returns**: 
+**Returns**:
 
-:: 
+::
 
 	First query:  [[2, 'banana', datetime.date(2019, 1, 1)], [3, 'carrot', datetime.date(2020, 1, 1)], [1, 'apple', datetime.date(2018, 1, 1)]]
 	Second query:  [[3, 'carrot', datetime.date(2020, 1, 1)], [4, 'donut', datetime.date(2021, 1, 1)]]
 
-.. note:: 
+.. note::
+
+	Multiple statement queries are not able to use placeholder values for parameterized queries.
+
+
+
+Server-side asynchronous query execution
+==========================================
+
+Server-side asynchronous query execution allows you to run a long query in the background while executing other asynchronous or synchronous queries. It is not possible to retrieve the results of a server-side asynchronous query, so these queries are best used for running lambda functions, or to run SELECTs for warming the cache.
```

Comment:
When I mentioned Lambda I was referring to AWS Lambda, which I don't think is the same thing as lambda functions necessarily. Maybe we should just generically say "DMLs" instead of lambda functions? Also, maybe it would be good to explicitly say that the benefit of server-side async is to be able to free up a connection or potentially even spin down an entire service (such as AWS Lambda) while a long-running database job is still underway.

Reply:
Oh, thanks for the clarity. I was going off memory of our conversation and that had begun to tread into areas which I don't have any real experience with, hence not understanding Lambda vs lambda (which I didn't think we provided).

Reply:
I did this and basically quoted you.